src/cell_lists/full_grid.jl

```diff
@@ -0,0 +1,117 @@
+"""
+    FullGridCellList(; min_corner, max_corner, search_radius = 0.0, periodicity = false)
+
+A simple cell list implementation where each (empty or non-empty) cell of a rectangular
+(axis-aligned) domain is assigned a list of points.
+This cell list only works when all points are inside the specified domain at all times.
+
+Use the default arguments to create an empty "template" cell list that can be used to create
```

Comment:
there are no default arguments for `min_corner` and `max_corner`?

Reply:
No, these are always mandatory. I've updated the docs to make that clear.